.clang-format

```diff
@@ -38,7 +38,9 @@ AlignConsecutiveMacros:
 
 # Newlines
 ColumnLimit:     0
-BreakBeforeBraces: Allman         # Braces on their own lines
```

Comment:
Please keep Allman braces.

Reply:
I removed that because it doesn't allow for single-line lambdas. Perhaps we could customize?

Reply:
I would advise against customizing; I don't think we want to open that can of worms again 😄. If the price we pay is not having single-line lambdas, so be it.